src/Nethermind/Nethermind.Serialization.Rlp/TxDecoder.cs

```diff
@@ -527,29 +597,52 @@ private int GetContentLength(T item, bool forSigning, bool isEip155Enabled = fal
         /// </summary>
         public int GetLength(T tx, RlpBehaviors rlpBehaviors)
         {
-            int txContentLength = GetContentLength(tx, false);
-            int txPayloadLength = Rlp.LengthOfSequence(txContentLength);
+            int txPayloadLength;
+
+            if (tx.Type is TxType.Blob)
```

Comment:
`tx.SupportsBlobs`?

Reply:
If there will be more transactions that support blobs, it can be of different length. So let the code stay non forward compatible.